README.md

```diff
@@ -1,35 +1,23 @@
-| [![QuizMaster - Monitoring API](https://github.com/FS-FAST-TRACK/QuizMaster/actions/workflows/backend.api.monitoring.yml/badge.svg)](https://github.com/FS-FAST-TRACK/QuizMaster/actions/workflows/backend.api.monitoring.yml)     | [![QuizMaster - Quiz API](https://github.com/FS-FAST-TRACK/QuizMaster/actions/workflows/backend.api.quiz.yml/badge.svg)](https://github.com/FS-FAST-TRACK/QuizMaster/actions/workflows/backend.api.quiz.yml)    |
-| [![QuizMaster - Quiz Session API](https://github.com/FS-FAST-TRACK/QuizMaster/actions/workflows/backend.api.quizsession.yml/badge.svg)](https://github.com/FS-FAST-TRACK/QuizMaster/actions/workflows/backend.api.quizsession.yml) | [![QuizMaster - Media API](https://github.com/FS-FAST-TRACK/QuizMaster/actions/workflows/backend.api.media.yml/badge.svg)](https://github.com/FS-FAST-TRACK/QuizMaster/actions/workflows/backend.api.media.yml) |
+[![QuizMaster - Account API](https://github.com/FS-FAST-TRACK/QuizMaster/actions/workflows/backend.api.account.yml/badge.svg)](https://github.com/FS-FAST-TRACK/QuizMaster/actions/workflows/backend.api.account.yml) | [![QuizMaster - Auth API](https://github.com/FS-FAST-TRACK/QuizMaster/actions/workflows/backend.api.auth.yml/badge.svg)](https://github.com/FS-FAST-TRACK/QuizMaster/actions/workflows/backend.api.auth.yml)
+---|---
+[![QuizMaster - Monitoring API](https://github.com/FS-FAST-TRACK/QuizMaster/actions/workflows/backend.api.monitoring.yml/badge.svg)](https://github.com/FS-FAST-TRACK/QuizMaster/actions/workflows/backend.api.monitoring.yml) | [![QuizMaster - Quiz API](https://github.com/FS-FAST-TRACK/QuizMaster/actions/workflows/backend.api.quiz.yml/badge.svg)](https://github.com/FS-FAST-TRACK/QuizMaster/actions/workflows/backend.api.quiz.yml)
+[![QuizMaster - Quiz Session API](https://github.com/FS-FAST-TRACK/QuizMaster/actions/workflows/backend.api.quizsession.yml/badge.svg)](https://github.com/FS-FAST-TRACK/QuizMaster/actions/workflows/backend.api.quizsession.yml) | [![QuizMaster - Media API](https://github.com/FS-FAST-TRACK/QuizMaster/actions/workflows/backend.api.media.yml/badge.svg)](https://github.com/FS-FAST-TRACK/QuizMaster/actions/workflows/backend.api.media.yml)
 
 ### Technology Stacks
-
-| Backend       | Frontend(Web) | Mobile       |
-| ------------- | ------------- | ------------ |
-| C#            | Next.js       | React-Native |
-| ASP.NET Core  | TailwindCSS   | NativeWind   |
-| Microservices |               |
-| API           |
-
-RabbitMQ
+Backend | Frontend(Web) | Mobile
+---|---|---
+C#|React|React-Native
+ASP.NET Core|Vite|NativeWind
```

Comment:
i think you forgot to pull from main master :D

Reply:
oh yeah, might change that again :'(